doc/api/util.md

```diff
@@ -68,8 +68,7 @@ exports.puts = util.deprecate(() => {
 It returns a modified function which warns once by default.
 
 If `--no-deprecation` is set then this function is a NO-OP.  Configurable
-at run-time through the `process.noDeprecation` boolean (only effective
-when set before a module is loaded.)
+at run-time through the `process.noDeprecation` boolean.
```

Comment:
@jasnell is that a replacement suggestion for the paragraph here? and the whole point of this PR is that the `*prior* to loading a module` bit didn’t really work, as far as I can tell.